lms/static/js/spec/student_account/account_settings_fields_spec.js

```diff
@@ -46,6 +46,56 @@ define(['backbone',
                 );
             });
 
+            it("update time zone dropdown after country dropdown changes", function () {
+                requests = AjaxHelpers.requests(this);
+                var selector = '.u-field-value > select';
+                var groups = '.u-field-value > select > optgroup';
+                var groupOptions = '.u-field-value > select > optgroup > option';
+                var normalOptions = '.u-field-value > select > option';
```

Comment:
For these more specialized selectors, could you construct them using the previous variables? For example, `groups` is `selector` with an additional string concatenated onto it, `groupOptions` has a similar relation to `groups`, etc. That way, if something changes in the html that defines `selector`, it only needs to be updated in one location.